etl_framework/config_mixins/FiltersMixin.py

```diff
@@ -29,6 +30,26 @@ def add_filters(self, filter_mappings):
             self.set_filter(getattr(filter_mappings, self.get_filter()))
             self.set_post_filter(getattr(filter_mappings, self.get_post_filter()))
 
+    def add_filters_from_module(self, filter_module):
```

Comment:
@ianlofs can we have `add_filters` call `add_filters_from_module`? Alternatively, we could add a `add_filters_from_module` method to teh AddFiltersMixin and call `super(AddFiltersMixin, self).add_filters_from_module`